hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/TestHddsDatanodeService.java

```diff
@@ -48,10 +48,7 @@
 import static org.apache.hadoop.hdds.HddsConfigKeys.HDDS_CONTAINER_TOKEN_ENABLED;
 import static org.apache.hadoop.ozone.OzoneConfigKeys.OZONE_SECURITY_ENABLED_KEY;
 import static org.apache.hadoop.ozone.OzoneConsts.OZONE_URI_DELIMITER;
-import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
Having the Junit 5 assert methods fully qualified is fine.  We can leave them in the file and prevent unneeded changes having to be merged.  We can leave the fully qualified imports for `assertFalse`, `assertTrue`, `assertNotNull`, `assertNull `and revert the changes in the file due to their omission.

Reply:
I know that it seems we are creating unnecessary diffs here but it's much cleaner having one import instead of five. Apart from the existing, we will add 2 more (`assertEquals`, `assertTrue`) and generate new diffs anyway.

```java
import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;
import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertNull;
import static org.junit.jupiter.api.Assertions.assertEquals;
```

Reply:
Let's just revert this for now.